domain/schema/model/sql/0020-unit.sql

```diff
@@ -313,3 +295,23 @@ SELECT
     u.password_hash
 FROM application AS a
 LEFT JOIN unit AS u ON a.uuid = u.application_uuid;
+
+CREATE TABLE unit_resolved (
+    unit_uuid TEXT NOT NULL PRIMARY KEY,
+    mode_id INT NOT NULL,
+    CONSTRAINT fk_unit_resolved_unit
+    FOREIGN KEY (unit_uuid)
+    REFERENCES unit (uuid),
+    CONSTRAINT fk_unit_resolved_mode
+    FOREIGN KEY (mode_id)
+    REFERENCES resolve_mode (id)
+);
```

Comment:
Keep in mind, that if we change this, then the watcher who watches the unit will also need to watch this table. That's not a bad thing persay, but we should be aware of it.

Reply:
Yeah I think this is probably better, since our watcher can be more specific